File: t/PACSFramework/CS/DicomHighLevelMWLScuDemo/UI/MyListView.cs
// *************************************************************
// Copyright (c) 1991-2019 LEAD Technologies, Inc.              
// All Rights Reserved.                                         
// *************************************************************
using System;
using System.Collections.Generic;
using System.Text;
using System.Windows.Forms;
using System.ComponentModel;
using System.Runtime.InteropServices;

namespace DicomDemo
{
    public class MyListView : ListView
    {       
        private List<ListViewItem> _StartedMPPS = new List<ListViewItem>();

        public List<ListViewItem> StartedMPPS
        {
            get { return _StartedMPPS; }
            set { _StartedMPPS = value; }
        }        
    }
}

File: t/PACSFramework/VB/DicomHighLevelMWLScuDemo/UI/MyListView.vb
' *************************************************************
' Copyright (c) 1991-2019 LEAD Technologies, Inc.              
' All Rights Reserved.                                         
' *************************************************************
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports System.Windows.Forms
Imports System.ComponentModel
Imports System.Runtime.InteropServices

Namespace DicomDemo
	Public Class MyListView : Inherits ListView
		Private _StartedMPPS As List(Of ListViewItem) = New List(Of ListViewItem)()

		Public Property StartedMPPS() As List(Of ListViewItem)
			Get
				Return _StartedMPPS
			End Get
			Set
				_StartedMPPS = Value
			End Set
		End Property
	End Class
End Namespace
